tests/recipes/recipe_lib_test.py

```diff
@@ -13,7 +14,7 @@ class BaseTestForMakeRecipe(RecipeCtx):
 
     recipe_name = None
     expected_compiler = (
-        "{android_ndk}/toolchains/llvm/prebuilt/linux-x86_64/bin/clang"
+        "{android_ndk}/toolchains/llvm/prebuilt/{system}-x86_64/bin/clang"
```

Comment:
So basically unit test was failing in osx? Maybe it could be a PR on its own with these fixes